torch/nn/parallel/replicate.py

```diff
@@ -111,6 +111,7 @@ def replicate(network, devices, detach=False):
         module_indices[module] = i
         for j in range(num_replicas):
             replica = module._replicate_for_data_parallel()
+            replica.is_replicated = True
```

Comment:
Maybe we can avoid introducing new attribute as replica already has a private attribute `_is_replica = True`
https://github.com/pytorch/pytorch/blob/b35cdc5200af963e410c0a25400fd07f30b89bca/torch/nn/modules/module.py#L1385